From ff2bab5bd1a57078d1529e0551973d0c2f3ebd8d Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Tue, 25 Aug 2015 16:03:42 +0800
Subject: [PATCH] MLK-11409-4: ARM: dts: imx7d-12x12-lpddr3-arm2.dts: disable
 ecspi3

The commit 3d2d705f6 enable ecspi3, that is not correct, because we use ecspi1
on arm2 board to support boot from spi-nor, correct it and add 'sleep' pin for
ecspi1.

Signed-off-by: Robin Gong <b38343@freescale.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index 747ad3b..496b47c 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -157,8 +157,9 @@
 &ecspi1 {
 	fsl,spi-num-chipselects = <1>;
 	cs-gpios = <&gpio4 19 0>;
-	pinctrl-names = "default";
+	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&pinctrl_ecspi1_1 &pinctrl_ecspi1_cs_1>;
+	pinctrl-1 = <&pinctrl_ecspi1_1 &pinctrl_ecspi1_cs_1>;
 	status = "disabled";
 
 	spi_flash1: m25p80@0 {
@@ -176,7 +177,7 @@
 	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&pinctrl_ecspi3_1 &pinctrl_ecspi3_cs_1>;
 	pinctrl-1 = <&pinctrl_ecspi3_1 &pinctrl_ecspi3_cs_1>;
-	status = "okay";
+	status = "disabled";
 
 	spi_flash3: m25p80@0 {
 		#address-cells = <1>;
-- 
1.8.0

